flink-table-store-core/src/main/java/org/apache/flink/table/store/file/compact/LazyCompactManager.java

```
@@ -36,13 +38,25 @@ public boolean shouldWaitCompaction() {
     }
 
     @Override
-    public void addNewFile(DataFileMeta file) {}
+    public void addNewFile(DataFileMeta file) {
+        wrapped.addNewFile(file);
+    }
+
+    @Override
+    public void triggerCompaction(boolean fullCompaction) {
+        if (fullCompaction) {
+            wrapped.triggerCompaction(true);
```

We can throw unsupported exception here.